src/Miunie.Core.XUnit.Tests/LanguageResourcesTests.cs

```diff
@@ -0,0 +1,62 @@
+using System;
+using Xunit;
+
+
+namespace Miunie.Core.XUnit.Tests
+{
+    public class LanguageResourcesTests
+    {
+        LanguageResources langResources;
+        DataStorageMock storage;

```

Comment:
`private` to be explicit and `readonly` to make sure you don't reinitialize accidentally.

Reply:
for both the mock and resources of course.